### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by marcokumu**
# Description

I worked on this issue #316 .
This contact form involved reusing the button component that had been created.

# Screenshot (optional)
Screenshot from 2023-02-27 17-42-52

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by marcokumu** [comment]
@IanOdhiambo9 kindly review

File: libs/pages/elewa/contact/src/lib/components/contact-form/contact-form.component.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<div class="contact-main">
+  <div class="contact-info">
+    <h1 class="heading">Contact</h1>
+    <p>
+      Looking for a quick answer to your question? Check out our FAQs. <br />
+      <br />
+      <br />
+      For everything else, drop us a line and we'll get back to you.
+    </p>
+    <div class="icons">
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/contact/src/lib/components/contact-form/contact-form.component.html:10]: Your design is great but still not quite there.
The anchor text for privacy policy is white not blue
The margin between input fields is not consistent
The width of the input fields is not consistent
No mobile design for responsiveness

File: libs/pages/elewa/contact/src/lib/components/contact-form/contact-form.component.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  message1 = "Send message"
+  contactForm: FormGroup;
+
+  constructor(private formBuilder: FormBuilder) {
+    this.contactForm = this.formBuilder.group({
+      name: '',
+      email: '',
+      companyName: '',
+      category: '',
+      message: '',
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/contact/src/lib/components/contact-form/contact-form.component.ts:18]: Move logic to a function "buildContactForm" and call function in ngOnInit

File: libs/pages/elewa/contact/src/lib/pages-elewa-contact.module.ts [hunk 3]
```diff
@@ -2,9 +2,19 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { ContactRoutingModule } from './contact.routing';
 import { ContactPageComponent } from './pages/contact-page/contact-page.component';
+import { ContactFormComponent } from './components/contact-form/contact-form.component';
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/contact/src/lib/pages-elewa-contact.module.ts:5]: order of imports is not correct

File: libs/pages/elewa/contact/src/lib/pages/contact-page/contact-page.component.html [hunk 4]
```diff
@@ -1 +1,3 @@
-<p>contact-page works!</p>
+
+<elewa-group-contact-form></elewa-group-contact-form>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/contact/src/lib/pages/contact-page/contact-page.component.html:2]: add the main template page here

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Should match provided design

**Comment by IanOdhiambo9** [comment]
Closing because no updates made here.

**Pull request closed without merging**